packages/auth/src/accessToken/routes.test.ts

```diff
@@ -202,9 +202,9 @@ describe('Access Token Routes', (): void => {
           client: TEST_CLIENT,
           jwk: testJwk
         } as ClientKey)
-      const now = new Date(new Date().getTime() + 4000)
-      jest.useFakeTimers()
-      jest.setSystemTime(now)
+      const now = new Date(new Date().getTime() + 4000 * 1000)
+      jest.useFakeTimers({ now })
+      // jest.setSystemTime(now)
```

Comment:
```suggestion
```

Reply:
Thought I saved the change before committing it 🤦